run/image-processing/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "nodejs-image-processing",
+  "version": "1.0.0",
```

Comment:
I think if the package is private, you don't need a version. Private means it can't be published.

Reply:
I think it's important for developers to think about version numbers for all projects even if not publishing, but I can't say I thought about this one. As a practice what do we prefer to do in samples on version and private?

Reply:
I agree that version numbers are important. In our samples though we never paid much attention to them and I'd prefer not having them than having meaningless numbers.